### ISSUE TITLE
Update deprecated configration

### ISSUE THREAD

**Issue by haroonshah2764** [systers/powerup-android#1084]
## Description
As developers
we are using deprecated configuration in app level `build.gradle` file. e.g. we are still using `compile` instead of `implementation`. We need to update the configrations
so that we are following the latest guidelines.

## Acceptance Criteria
- [ ] Update the deprecated configurations.
### Enhancement to Update [Optional]
- [ ] Configrations

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
1 hour

**Comment by haroonshah2764** [comment]
Can I send a PR to fix this issue?

**Comment by pprathameshmore** [comment]
@haroonshah2764 Yeah! We must update the configuration! The latest Android Gradle plugin now requires you to explicitly define. [Reference](https://stackoverflow.com/questions/44493378/whats-the-difference-between-implementation-and-compile-in-gradle)

**Comment by Rimjhim28** [comment]
Sounds good! @haroonshah2764 Can you please initiate a discussion for this on slack so that we can have different opinions?

**Comment by geekanamika** [comment]
@rimjhim28 Android studio 3.1 also shows the warning in `Build` section due to `compile`. Since this is a small fix, @haroonshah2764 Go ahead.
